cli.ts

```diff
@@ -76,5 +95,6 @@ async function cli(args: ParsedArgs) {
 }
 
 cli(parseArgs(process.argv.slice(2))).catch((e) => {
+  // TODO: handle http errors
```

Comment:
The error here was shown like `[object Object]` when I got API error because of invalid access token caused by a bug in `lib/interact-for-workspace-id.ts` which I fixed in this PR

Reply:
i fixed this if you pull main